[![Review Assignment Due Date](https://classroom.github.com/assets/deadline-readme-button-22041afd0340ce965d47ae6ef1cefeee28c7c493a6346c4f15d667ab976d596c.svg)](https://classroom.github.com/a/aKWLU3-A)

Group usernames: linaidan, aurelia-chiara, Samirainnocente

"How our technology works" (Sequencing Based Spatial Transcriptomics)

Sequencing based spatial transcriptomics is a kind of RNA sequencing that combines the quantification of gene expression profiles while keeping spatial context. The tissue and its mRNA can be analyzed in situ without the loss of their spatial information. This allows researchers to study tissues and measure the cells' transcripts and their position. The spatial information is important to understand various processes
in biological systems such as gene expression, cell signalling and intercellular communication. The method usually consists of preparing the tissue slides and capturing the mRNA with specific barcodes that also provide the positional coordinates. Then the captured mRNAs are reverse transcribed and sequenced using
high-throughput sequencing. The sequencing reads are then used to create a spatial map of the gene expression across the tissue. 

In general Spatial Transcriptomics can be divided into three categories: In situ hybridization (mostly referred to as sequencing-based ST), in situ sequencing and in situ capturing technologies. The first two methods are used to target specific genes a priori, when the genes of interest are already known. The latter technique can be used as an exploratory approach to discover new patterns.




